TACC/protx-dashboard
# Task/wg 169 maptiles service update

## Overview: ##

Updated the MapProviders in the protx dashboard to drop all Stamen Maptiles and ins5tead use OpenStreetMaps free services (until we can implement our own TACC Tiles).

## Related Jira tickets: ##

* [WG-169](https://jira.tacc.utexas.edu/browse/WG-69)

## Summary of Changes: ##

- Dropped all of the Stamen provider sources.
- Verified the OSM providers are still accurate.
- Merged in all upstream changes from `main`.

## Testing Steps: ##

1. Build project and run dashboard.
2. Make sure you get maptiles in leaflet from OSM.

## UI Photos:



## Notes: ##
